### ISSUE TITLE
3.3.0 not installing anymore

### ISSUE THREAD

**Issue by JustinAiken** [neo4jrb/neo4j-rake_tasks#42]
- `http://dist.neo4j.org/neo4j-3.3.0-unix.tar.gz` now returns a 403
- But it does seem to be available at `https://neo4j.com/artifact.php?name=neo4j-community-3.3.0-unix.tar.gz`

**Comment by cheerfulstoic** [comment]
Oy... I think they've been switching things around.  Does 3.3.1 work?  I ask because I just installed 3.3.1 via the rake tasks about 20 minutes ago and it worked for me...

**Comment by JustinAiken** [comment]
Yeah, 3.3.1 still installs... guess they just moved the "old" 3.3.x ones... guessing other 3.2.x and the such also moved..

Unfortunately, 3.3.1 breaks on our CI...  `NEO4J_dbms_connector_http=some_other_port` worked in 3.3.0 but not 3.3.1, but that's another issue..

**Comment by cheerfulstoic** [comment]
🙁  Weird.

You could certainly just download and extract the archive yourself and run the binaries.  This gem is really here primarily to make it easy for new users to get up and running with Neo4j in Ruby, though that's becoming a lot less important with the release of the Neo4j Desktop app (and honestly I'd like to support as little code as possible)

But it would be good to know if Neo4j is officially changing downloads paths or if the one version is just a fluke (shout out to @ryguyrg or @jexp who might know or at least might know who would know)
